static/js/panels/ProfileForm.tsx

```diff
@@ -11,6 +11,49 @@ import { createProfile } from "../api/profiles";
 import { useQueryClient } from "@tanstack/react-query";
 import { queryKeys } from "../util/queryKeys";
 import SubmitButton from "../buttons/SubmitButton";
+import NetworkSelector from "../components/NetworkSelector";
+import { LxdDevices } from "../types/device";
+import StorageSelector from "../components/StorageSelector";
+import CpuLimitSelector from "../components/CpuLimitSelector";
+import CloudInitConfig from "../components/CloudInitConfig";
+import { LxdConfigPair } from "../types/config";
+import { LxdProfile } from "../types/profile";
+import MemoryLimitSelector from "../components/MemoryLimitSelector";
+import { CpuLimit, CPU_LIMIT_TYPE } from "../types/limits";
+import {
+  DEFAULT_CPU_LIMIT,
+  DEFAULT_DISK_DEVICE,
+  DEFAULT_MEM_LIMIT,
+  DEFAULT_NIC_DEVICE,
+} from "../util/defaults";
+
+const getCpuLimit = (cpuLimit: CpuLimit) => {
+  switch (cpuLimit.selectedType) {
+    case CPU_LIMIT_TYPE.DYNAMIC:
+      if (cpuLimit.dynamicValue) {
+        return `${cpuLimit.dynamicValue}`;
+      }
+      return null;
+    case CPU_LIMIT_TYPE.FIXED_RANGE:
+      if (!cpuLimit.rangeValue) return null;
+      if (
+        cpuLimit.rangeValue.from !== null &&
+        cpuLimit.rangeValue.to !== null
+      ) {
+        return `${cpuLimit.rangeValue.from}-${cpuLimit.rangeValue.to}`;
+      }
+      return null;
+    case CPU_LIMIT_TYPE.FIXED_SET:
+      if (cpuLimit.setValue) {
+        if (cpuLimit.setValue.length > 1) {
```

Comment:
I think we need to look for commas, because the value could be a multiple digit one.

Reply:
Ok, is it ok to keep it simple, without crazy validation through regex? (which I'm trying to avoid, at least at this stage)

Reply:
Done @ [86a81f1](https://github.com/canonical/lxd-ui/commit/86a81f106d7d3fceca7bd45be2f5972edebc365a)

Reply:
Yeah I think this is fine.